Common++/src/IpAddress.cpp

```diff
@@ -43,6 +46,29 @@ namespace pcpp
 	}
 
 
+	bool IPv4Address::matchSubnet(const std::string& subnet) const
+	{
+		std::stringstream ss(subnet);
+		std::string subnetOnly, subnetPrefixStr;
+		std::getline(ss, subnetOnly, '/');
+		std::getline(ss, subnetPrefixStr);
+
+		uint32_t subnetPrefix = std::stoi(subnetPrefixStr);
```

Comment:
`std::stoi` throws a [`std::invalid_argument`](https://en.cppreference.com/w/cpp/error/invalid_argument) exception if no conversion could be performed  (https://en.cppreference.com/w/cpp/string/basic_string/stol), maybe we should surround it with `try...catch` to prevent this exception which might be difficult for users to understand?

Reply:
Done